doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run `VxWorks`_ as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to VxWorks
+***********************
+
+VxWorks®, the industry’s leading real-time operating system, delivers all the performance, determinism, reliability,
+safety, and security capabilities you need to meet the highest standards for running the computing systems of the most
+important critical infrastructure.
+
+Steps for Using VxWorks as User OS
+**********************************
+
+#. Build VxWorks
+
+   Follow the `VxWorks Getting Started Guide <https://docs.windriver.com/bundle/vxworks_7_tutorial_kernel_application_workbench_sr0610/page/rbu1422461642318.html>`_
+   to setup the VxWorks development environment and build the VxWorks Image.
+
+   .. note::
+      The following kernel configuration should be excluded:
```

Comment:
Let's make the excluded and included words bold:
```suggestion
      The following kernel configuration should be **excluded**:
```

Reply:
Will fix it.